lawnchair/res/values/config.xml

```diff
@@ -18,6 +18,18 @@
 <resources>
     <string name="app_filter_class" translatable="false">app.lawnchair.DefaultAppFilter</string>
     <string name="calendar_component_name" translatable="false">com.google.android.calendar/com.android.calendar.AllInOneActivity</string>
+    <string-array name="dynamic_calendar_components_name">
+        <item>com.android.calendar</item>
+        <item>com.xiaomi.calendar</item>
+        <item>com.samsung.android.calendar</item>
+        <item>com.blackberry.calendar</item>
+        <item>com.huawei.calendar</item>
+        <item>com.simplemobiletools.calendar.pro</item>
+        <item>org.lineageos.etar</item>
+        <item>ws.xsoh.etar</item>
+        <item>org.withouthat.acalendarplus</item>
+        <item>org.withouthat.acalendar</item>
+    </string-array>
```

Comment:
Need to resort? And you can also add `com.oneplus.calendar` to handle the Oneplus calendar.